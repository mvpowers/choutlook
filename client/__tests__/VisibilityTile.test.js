import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import VisibilityTitle from '../src/components/VisibilityTitle';

Enzyme.configure({ adapter: new Adapter() });

describe('VisibilityTile', () => {
  it('should be defined', () => {
    expect(VisibilityTitle).toBeDefined();
  });
  it('should render correctly', () => {
    const tree = shallow(<VisibilityTitle />);
    expect(tree).toMatchSnapshot();
  });
});
