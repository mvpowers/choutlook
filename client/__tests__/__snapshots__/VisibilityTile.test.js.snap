// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VisibilityTile should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "active": true,
      "children": [Function],
      "containment": null,
      "delayedCall": false,
      "intervalCheck": true,
      "intervalDelay": 100,
      "minTopValue": 0,
      "offset": Object {},
      "partialVisibility": false,
      "resizeCheck": false,
      "resizeDelay": 250,
      "resizeThrottle": -1,
      "scrollCheck": false,
      "scrollDelay": 250,
      "scrollThrottle": -1,
    },
    "ref": null,
    "rendered": [Function],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "active": true,
        "children": [Function],
        "containment": null,
        "delayedCall": false,
        "intervalCheck": true,
        "intervalDelay": 100,
        "minTopValue": 0,
        "offset": Object {},
        "partialVisibility": false,
        "resizeCheck": false,
        "resizeDelay": 250,
        "resizeThrottle": -1,
        "scrollCheck": false,
        "scrollDelay": 250,
        "scrollThrottle": -1,
      },
      "ref": null,
      "rendered": [Function],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
